Bottom: eb8651cae2fb09aca0ff9ab019428fd724935bff
Top:    36d9f876882b4c1a4eba407c21eefcebdecec2f2
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-22 20:53:52 -0400

Refresh of wm9712-register

---

diff --git a/sound/soc/fsl/dspeak01-audio-fabric.c b/sound/soc/fsl/dspeak01-audio-fabric.c
index 491100e..07b44b1 100644
--- a/sound/soc/fsl/dspeak01-audio-fabric.c
+++ b/sound/soc/fsl/dspeak01-audio-fabric.c
@@ -145,7 +145,6 @@ static __init int dspeak01_fabric_init(void)
 	if (!machine_is_compatible("digispeaker,dspeak01"))
 		return -ENODEV;
 
-	printk("Initializing Digispeaker audio\n");
 	card.platform = &mpc5200_audio_dma_platform;
 	card.name = "Efika";
 	card.dai_link = dspeak01_fabric_dai;
diff --git a/sound/soc/fsl/efika-audio-fabric.c b/sound/soc/fsl/efika-audio-fabric.c
index 58121a5..5126a81 100644
--- a/sound/soc/fsl/efika-audio-fabric.c
+++ b/sound/soc/fsl/efika-audio-fabric.c
@@ -55,8 +55,6 @@ static __init int efika_fabric_init(void)
 	if (!machine_is_compatible("bplan,efika"))
 		return -ENODEV;
 
-
-	printk("Initializing Efika audio\n");
 	card.platform = &mpc5200_audio_dma_platform;
 	card.name = "Efika";
 	card.dai_link = efika_fabric_dai;
diff --git a/sound/soc/fsl/mpc5200_psc_ac97.c b/sound/soc/fsl/mpc5200_psc_ac97.c
index 00e832f..ba8e114 100644
--- a/sound/soc/fsl/mpc5200_psc_ac97.c
+++ b/sound/soc/fsl/mpc5200_psc_ac97.c
@@ -322,6 +322,9 @@ static int __devinit psc_ac97_of_probe(struct of_device *op,
 	if (rc != 0)
 		return rc;
 
+	for (i = 0; i < ARRAY_SIZE(psc_ac97_dai); i++)
+		psc_ac97_dai[i].dev = &op->dev;
+
 	rc = snd_soc_register_dais(psc_ac97_dai, ARRAY_SIZE(psc_ac97_dai));
 	if (rc != 0) {
 		pr_err("Failed to register DAI\n");
diff --git a/sound/soc/fsl/pcm030-audio-fabric.c b/sound/soc/fsl/pcm030-audio-fabric.c
index 83ebfa5..4bd957a 100644
--- a/sound/soc/fsl/pcm030-audio-fabric.c
+++ b/sound/soc/fsl/pcm030-audio-fabric.c
@@ -55,8 +55,6 @@ static __init int pcm030_fabric_init(void)
 	if (!machine_is_compatible("phytec,pcm030"))
 		return -ENODEV;
 
-
-	printk("Initializing Phytec audio\n");
 	card.platform = &mpc5200_audio_dma_platform;
 	card.name = "pcm030";
 	card.dai_link = pcm030_fabric_dai;
